compiler/testData/diagnostics/tests/suspendConversion/suspendConversionCompatibilityInDisabledMode.fir.kt

```diff
@@ -11,7 +11,7 @@ object Test1 {
 
         fun test() {
             val result = foo(::bar)
-            <!DEBUG_INFO_EXPRESSION_TYPE("kotlin.Int")!>result<!>
+            <!DEBUG_INFO_EXPRESSION_TYPE("kotlin.String")!>result<!>
```

Comment:
Note that test changes under `...Disabled*` are regression, since they should not be converted. Once we add feature flag look-up, these can be reverted.